services/dy-3dvis/docker-compose.devel.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.4'
+services:
+  simcore-paraviewweb:
+    build:
+      target: development
+    environment: 
+      - S3_ENDPOINT=${S3_ENDPOINT}
+      - S3_ACCESS_KEY=${S3_ACCESS_KEY}
+      - S3_SECRET_KEY=${S3_SECRET_KEY}
+      - S3_BUCKET_NAME=${S3_BUCKET_NAME}
+      - POSTGRES_ENDPOINT=${POSTGRES_ENDPOINT}
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
+      - TEST_DATA_PATH=/home/root/test-data
+    depends_on:
+      - postgres
+      - minio
+    volumes: 
+      - ./simcoreparaviewweb/src:/home/root/src
+      - ../../packages/simcore-sdk/src:/home/root/packages/packages/simcore-sdk/src
+      - ../../packages/s3wrapper/src:/home/root/packages/packages/s3wrapper/src
+      - ./test-data:/home/root/test-data
+  postgres:
+    image: postgres:10
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
+    ports:
+      - "5432:5432"
+  #--------------------------------------------------------------------
+  adminer:
+    image: adminer
+    ports:
+      - 18080:8080
+    depends_on:
+      - postgres
+  #--------------------------------------------------------------------
+  minio:
+    image: minio/minio
+    environment:
+      - MINIO_ACCESS_KEY=${S3_ACCESS_KEY}
+      - MINIO_SECRET_KEY=${S3_SECRET_KEY}
+    ports:
+      - "9001:9000"
+    command: server /data
```

Comment:
Same here. Can we have command line arguments running only specific parts of the platform stack directly from the root docker-compose?

Reply:
well basically, it is not part of the platform per se so I am not convinced this belongs with the root docker-compose file.
I use these devel compose to create a platform free testing environment that is faster to set-up (it automatically uploads whatever is necessary and reduces manually settings things). I think some of these things could be shared among services as it is probable that having a minimal S3,DB config will be necessary for a lot of services.